### DISCUSSION TITLE
LNURL/Lightning Address not working

### DISCUSSION THREAD

**Discussion by doccodez** [btcpayserver/btcpayserver#3063] [Technical Support]
I have added turned on LNURL, and added lightning addresses but they do not work. I know they work using GET requests so here is the error I am getting on the server. 

[error] 2623#2623: *1782 open() "/var/www/html/.well-known/lnurlp/xxxxx" failed (2: No such file or directory)

xxxxx being the username used for the lightning address.
Any idea on how to fix this?

**Comment by NicolasDorier** [comment]
we are aware of some people experiencing this issue. Trying to find out where it comes from.
As for your specific error above, this is expected, the response is returned by btcpayserver, this is not a static file.

**Reply by doccodez** [reply]
Oh okay good to know it's expected it was just the only error I could find. I believe the lightning address was working before I set btcpayserver up behind a reverse proxy so not sure whether that's the problem? @NicolasDorier

**Reply by pavlenex** [reply]
can you update @cubanhodl? 1.3.3 may have fixed the issue.

**Reply by doccodez** [reply]
> can you update @cubanhodl? 1.3.3 may have fixed the issue.

I have updated to 1.3.3 and the issue still persists. @pavlenex
